### ISSUE TITLE
Wrong URL returned for `eos.url.xroot` attribute

### ISSUE THREAD

**Issue by amadio** [cern-eos/eos#41]
During the development of the redirection to XRootD feature in ROOT (https://github.com/root-project/root/pull/11644), I noticed that sometimes the URL returned by EOS for a file is only the management URL instead of the full URL. The problem can be easily reproduced by just copying a file into EOS and calling `getfattr` on it:
```bash
epsftws ~ $ ssh lxplus
lxplus ~ $ cd /eos/home-a/amadio
lxplus amadio $ cp ~/.bashrc bashrc
lxplus amadio $ getfattr -n eos.url.xroot bashrc
# file: bashrc
eos.url.xroot="root://eoshome-a.cern.ch/"

lxplus amadio $ getfattr -n eos.url.xroot .
# file: .
eos.url.xroot="root://eoshome-a.cern.ch//eos/user/a/amadio/"

lxplus amadio $ getfattr -n eos.url.xroot bashrc
# file: bashrc
eos.url.xroot="root://eoshome-a.cern.ch//eos/user/a/amadio/bashrc"
```

**Comment by esindril** [comment]
Hi Guilherme,

It's more efficient if you open such bug reports to the internal Jira tracker. This github issue is mostly for external people who don't have access to our Jira. I will more this once to our Jira.

Thanks,
Elvin

**Comment by apeters1971** [comment]
commit 8fa602d202dd4fb29d35eb014eff5764f1b40f91
Author: Andreas Joachim Peters <andreas.joachim.peters@cern.ch>
Date:   Wed Nov 30 11:12:06 2022 +0100

    FUSEX: have the xroot url available available after file creation - fixes GITHUB #41
    
    
   
The origin was that after creation the server sends back the latest MD object with the assigned clock value. The MD copy operation was not preserving the fullpath stored already in the local MD object.
